nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -310,6 +313,7 @@ def apply(
         dataset: Optional[Dataset] = None,
     ) -> TModel:
         self._set_backend_entity(model)
+        # nodes_to_compress includes nodes from the ignored scope to be added to bitwidth_distribution_str
         nodes_to_compress = self._get_nodes_to_compress(graph)
```

Comment:
I'd rename it to avoid confusion with actual list of nodes to compress
```suggestion
        candidates_to_compress = self._get_nodes_to_compress(graph)
```